site-src/contributing/gep.md

```diff
@@ -0,0 +1,115 @@
+# Gateway Enhancement Proposal (GEP)
+
+Gateway Enhancement Proposals (GEPs) will serve a similar purpose to
+the [KEP][kep] process for the main Kubernetes project:
+
+1. Ensure that changes to the API follow a known process and discussion
+  in the OSS community.
+1. Make changes and proposals discoverable (current and future).
+1. Document design ideas, tradeoffs, decisions that were made for
+  historical reference.
+
+As this is still a relatively new project, we don't wish to encumber
+proposals with a large amount of boilerplate process. The most
+important part of the GEP are the last two items: making sure that we
+can easily find previous discussions and alternatives considered.
+
+## Process
+
+### New GEP
+
+* File an issue that will be used to track the GEP progress. Use the existing
+  issue templates. Use this number for naming the GEP (i.e. issue #123 would be
+  named `gep-123.md`).
+* Discussion and comments occur on the issue and shared docs. Any
+  relevant shared documents should be linked off of the Github issue.
+  * Create a GEP document in `site-srcs/gep/gep-<issue
+    #>-short-description.md` with status `proposed`; the sections that
+    MUST be kept up to date are `TLDR` and `References`. Other
+    sections can refer to the issue or shared doc temporarily.
+  * We should try to resolve most of the discussion in this phase as doing the
+    discussion on a GEP is not very easy (at the moment) from a workflow
+    perspective.
+* When there is consensus on the approach, consolidate any content,
+  links into the document (see template format below). Submit the
+  PR for merging.
+* Propagate iterations on the GEP proposal from the issue/shared docs
+  to the GEP.
+* When there are no further comments, update GEP with status
+  "accepted". This means that we are committing to implementing/have
+  implemented the feature.
+  * Make sure any shared docs are going to be in a format that is accessible in
+    the future. If possible, copy the contents of the document into the GEP
+    along with comments.
+* Set the GEP status (accepted, implemented etc).
+* Tracking issue should be marked as closed.
+
+## Format
+
+``` markdown
+# GEP-1234: Add Chaosnet protocol support
+
+Status: (accepted|implemented|withdrawn|replaced)
+
+(See definitions in: https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/kep.yaml#L9)
```

Comment:
A more accurate description is this:
https://github.com/kubernetes/enhancements/blob/master/keps/sig-architecture/0000-kep-process/README.md#kep-workflow

What confused me is that `accepted` is not a valid status. Can we please correct the link here or update line 52 (and content) accordingly to not use `accepted`?

Reply:
Good point, maybe that should be `implementable` instead to more closely mirror upstream terminology. I don't think we need to support all of the potential states from upstream though.

Reply:
I thought I took all of the values from upstream. Let me fix this.